R/06-simplegeom-obj.R

```diff
@@ -109,3 +114,12 @@ setAs("data.frame", "simplegeom", function(from) {
   simplegeom <- new("simplegeom", Srl, proj4string = CRS("+proj=longlat +datum=WGS84"))
   return(simplegeom)
 })
+
+#'@importFrom sp Polygons Polygon CRS proj4string
+setAs("SpatialPolygons", "simplegeom", function(from) {
+  
+  stopifnot(grepl('datum=WGS84', proj4string(from)))
+  
+  simplegeom <- new("simplegeom", from@polygons, proj4string = CRS("+proj=longlat +datum=WGS84"))
```

Comment:
Should the datum for this come from the proj4string, so that others can be used in the future?  Then other datums can just be added as 'ors' to the grepl?